openmls/src/group/mls_group/mod.rs

```diff
@@ -406,10 +406,20 @@ impl MlsGroup {
         &self.group
     }
 
-    /// Clear the pending proposals.
-    #[cfg(test)]
-    pub(crate) fn clear_pending_proposals(&mut self) {
-        self.proposal_store.empty()
+    /// Clear the pending proposals, if the proposal store is not empty.
+    ///
+    /// Warning: Once the pending proposals are cleared it will be impossible to process
+    /// a Commit message that references proposals those proposals. Only use this
+    /// function as a last resort, e.g. when a call to
+    /// `MlsGroup::commit_to_pending_proposals` fails.
+    pub fn clear_pending_proposals(&mut self) {
```

Comment:
Can you please move this up to the block of public functions? It is still in the block of test functions

Reply:
8b9cc80